src/schema/generated-schema-types.ts

```diff
@@ -3368,24 +3404,24 @@ export type QueryShareInfoArgs = {
  * - [[SOAP XML-to-JSON Documentation]](https://wiki.zimbra.com/wiki/Json_format_to_represent_soap)
  */
 export type QuerySearchArgs = {
-  contact?: Maybe<Scalars['String']>,
-  cursor?: Maybe<Cursor>,
-  fetch?: Maybe<Scalars['String']>,
-  fullConversation?: Maybe<Scalars['Boolean']>,
-  limit?: Maybe<Scalars['Int']>,
-  needExp?: Maybe<Scalars['Boolean']>,
-  memberOf?: Maybe<Scalars['Boolean']>,
-  offset?: Maybe<Scalars['Int']>,
-  query?: Maybe<Scalars['String']>,
-  recip?: Maybe<Scalars['Int']>,
-  sortBy?: Maybe<SortBy>,
-  types?: Maybe<SearchType>,
-  resultMode?: Maybe<Scalars['String']>,
-  inDumpster?: Maybe<Scalars['Boolean']>
-};
-
-
-/** 
+  contact?: Maybe<Scalars['String']>;
```

Comment:
i think you run into conflicts due to this

Reply:
Wish me luck then ;)